src/sample.py

```diff
@@ -20,6 +20,7 @@ def tounicode(s, encoding='utf-8'):
 
 fontdata = open (sys.argv[1], 'rb').read ()
 text = tounicode(sys.argv[2])
+codepoints = list(map(ord, text))
```

Comment:
on narrow Python 2 builds, ord() will fail if input text contains unicode characters beyond 0xFFFF.
The fontTools.misc.py23 has a workaround for that.